public/app/index.ts

```diff
@@ -7,5 +7,9 @@ if ((window as any).public_cdn_path) {
   __webpack_public_path__ = (window as any).public_cdn_path;
 }
 
+// This is a path to the public folder without '/build'
+(window as any).__grafana_public_path__ =
+  __webpack_public_path__.substring(0, __webpack_public_path__.lastIndexOf('build/')) || __webpack_public_path__;
+
```

Comment:
@torkelo I moved this here because it is also needed for the monaco loader.  Any better ideas than `(window as any)`?

Reply:
`global` i think gets transpiled to window